### ISSUE TITLE
Not sure

### ISSUE THREAD

**Issue by acedra** [Ultimaker/Cura#13064]
### Application Version

5.1.0

### Platform

Windows 11

### Printer

Ender 3 Pro

### Reproduction steps

Loaded original stl file. Chose selections for settings. tried to slice.

### Actual results

generated error. suggested posting bug

### Expected results

file to be sliced.

### Checklist of files to include

- [X] Log file
- [X] Project file

### Additional information & file uploads

[cura.log](https://github.com/Ultimaker/Cura/files/9378539/cura.log)
[bug.zip](https://github.com/Ultimaker/Cura/files/9378547/bug.zip)

**Comment by GregValiant** [comment]
Thanks for the report.
It sliced right away for me.  Someone from the Cura Team will take a look at the log file.

**Comment by Vandresc** [comment]
I also get the slicing error with the attached project, I will bring it up with the rest of the team.  Thank you for the feedback.

**Comment by rburema** [comment]
Tested the initial .zip the OP posted on an early alpha build of 5.2 -- slicing succeeds now. However, I'm on Linux (at the moment). Not closing yet, as @GregValiant may also be on Linux.

@Vandresc -- Where you on Windows when you tested this? Can you try with one of the 5.2 (later) alpha builds?

**Comment by GregValiant** [comment]
@rburema I'm running Windows 10.

**Comment by Vandresc** [comment]
@rburema yes, on Windows, and I can still reproduce with the latest alpha of 5.2.

**Comment by rburema** [comment]
Ok, let's make a separate internal ticket then -- maybe we should attach the others as well if they're still failing on Windows.

**Comment by jellespijker** [comment]
@rburema can you change the title of this issue in a more descriptive one?

**Comment by acedra** [comment]
Sorry about that folks. First time submitting a bug to anything so I didn't
know what format to follow.

**Comment by rburema** [comment]
@acedra That's fine, we get that a lot. I think @jellespijker meant for me to change the title.

**Comment by jellespijker** [comment]
@acedra we welcome these reports, so please keep on submitting any issue you find or feature requests you have.
